.github/actions/setup-python-build-env/action.yml

```diff
@@ -0,0 +1,11 @@
+name: "Setup Build Environment"
+description: "Install everything needed to build"
+
+runs:
+  using: "composite"
```

Comment:
I'm not too familiar with GH Actions. Can you tell me what this line does?

Reply:
I'm not sure if it is strictly needed as there is only one run command in this action. 
In general I think it is need combine multiple run statements into a single action 
https://wallis.dev/blog/composite-github-actions